packages/ember-resolver/lib/initializers.js

```diff
@@ -4,7 +4,9 @@
   Ember.Application.initializer({
     name: 'container-debug-adapter',
 
-    initialize: function(container, app) {
+    initialize: function() {
+      var app = arguments[1] || arguments[0];
+      var container = arguments[1] ? arguments[0] : app;
```

Comment:
Every version of Ember.Application since 1.0.0 has supported `.register` and `.inject`, this was the only container related public API that existed in 1.0.0.